### PR TITLE
/telco content refresh

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied content refresh as per [doc](https://docs.google.com/document/d/1a3otoXCbXtddJG2HCnYIIHJiDxvG-pzFPtLobVB4-vw/edit)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13395.demos.haus//telco
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6454

### CODE REVIEW

**Comment by mtruj013** [comment]
Ready for you @Sophie-32 !

**Comment by Sophie-32** [comment]
@mtruj013 - I tagged you in the copydoc on some small items.

File: templates/telco/index.html [hunk 1]
```diff
@@ -437,7 +500,7 @@ <h3 class="p-muted-heading">
         <div class="p-logo-section__item">
           {{ image (
             url="https://assets.ubuntu.com/v1/df155ef0-magma.png",
```
Comment (carkod) [inline templates/telco/index.html:502]: This logo was requeted to be removed I think?

File: templates/telco/index.html [hunk 2]
```diff
@@ -1,42 +1,61 @@
...
-      </p>
-      <p class="u-sv3">
-      Scale and complexity brings similar challenges, big or small. Hybrid cloud, public cloud, on-prem only large datacenter or highly distributed infrastructure - Canonical Open Source technology building blocks provide a unified approach to enabling Service Providers to meet any current or future use-cases, from OpenRAN, next generation Core (5G and beyond) or AI at the edge.
-      </p>
+    <div class="col-7 u-vertically-center">
+      <h1>Carrier-grade open source for telecom</h1>
+      <p class="p-heading--3">Canonical offers secure, modular and interoperable solutions for public and private clouds, multi-cloud or hybrid-cloud environments at centralised or distributed sites.</p>
       <p>
-        <a href="/telco/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
+        <a href="/telco/contact-us" class="p-button--positive">Contact us</a>
```
Comment (carkod) [inline templates/telco/index.html:16]: The CTA at the bottom is linking to the modal, it looks a bit inconsistent that this one doesn't open the modal?

**Review by carkod — APPROVED** [review]
Just the two logos I guess. Else, LGTM

File: templates/telco/index.html [hunk 3]
```diff
@@ -461,7 +524,7 @@ <h3 class="p-muted-heading">
         <div class="p-logo-section__item">
```
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/telco/index.html]: I tried a couple different versions from the asset server too, but with the "has-misaligned-logos" class that these old pages need for the logo section, all of them ended up looking super small like that unfortunately

File: templates/telco/index.html (hunk 2 above)
Comment (mtruj013) [reply templates/telco/index.html:16]: Had another look at the doc and it's actually not supposed to link to the modal at the bottom either, so I've updated

**Pull request merged**